probeinterface/wiring.py

```diff
@@ -13,7 +13,14 @@
         47, 46, 45, 44, 43, 42, 41, 40, 39, 38, 37, 36, 35, 34, 33, 32,
         31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16,
         15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0,
-        63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51, 50, 49, 48]
+        63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51, 50, 49, 48],
+    
+    # nicely given by Aaron Wrong
+    'ASSY-77> Adpt.A64-Om32_2x-sm >RHD2164' :[
```

Comment:
```suggestion
    'ASSY-77>Adpt.A64-Om32_2x-sm>RHD2164' :[
```

Reply:
oups I forgot to push my latest commit. I was fixed already.

Reply:
The lines provided wouldn't run, but I ran this instead
```
import probeinterface as pi
from probeinterface import plotting
probe = pi.get_probe('cambridgeneurotech','ASSY-77-H3')
probe.wiring_to_device('ASSY-77>Adpt.A64-Om32_2x-sm>RHD2164')
plotting.plot_probe(probe, with_contact_id=True, with_device_index=True)
```
The correspondence between probe site id and device channel id is good. However, I noticed an error on the ASSY-77-H3.json in the probeinterface_library. I have created an adjusted file and a pull request at GIN. 

P.S. There seems to be another issue posted there concerning ASSY-77-H6. Maybe it's worth double checking all of them...?

> 
> 
> See #56
> @Enakcihc could you have a look and test with this plot
> 
> ```python
> probe = get_probe('XXXXXXXXX')
> wire_probe(probe, 'ASSY-77>Adpt.A64-Om32_2x-sm>RHD2164')
> plot_probe(probe, with_contact_id=False, 
>                 with_device_index=False)
> ```